instrumentation/lettuce/lettuce-5.1/library/src/main/java/io/opentelemetry/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java

```diff
@@ -313,6 +315,16 @@ public synchronized Tracer.Span tag(String key, String value) {
         argsString = value;
         return this;
       }
+      if (key.equals("db.namespace") && SemconvStability.emitOldDatabaseSemconv()) {
+        // map backwards into db.redis.database.index
+        int val = Integer.parseInt(value);
```

Comment:
could use `long val = Long.parseLong(value);` here to get rid of the cast when setting the attribute